curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641daabed8d0584b1150c953.md

```diff
@@ -0,0 +1,172 @@
+---
+id: 641daabed8d0584b1150c953
+title: Step 28
+challengeType: 0
+dashedName: step-28
+---
+
+# --description--
+
+Now it's time to extract the part of the `bio` for uniformity. If the `bio` text is more than 50 characters, then you should extract the first 50 and replace the rest with an ellipsis. The `slice` method is ideal for this use case. Apart from using it to extract the part of an array you want, you can also use it on strings to do the same thing.
+
+You might be thinking you can use `if` statement here, but you can't put an `if` statement inside a template literal. What you have to use instead is a ternary operator, AKA one line `if...else` statement.
+
+Extract the first 50 letters of the authors' `bio` text and replace the rest with ellipsis (...). For the else statement, use the `bio` itself.
```

Comment:
```suggestion
Some of the author bios are much longer than others. To give the cards a uniform look, you can extract the first 50 characters of each one and replace the rest with an ellipsis (...). Otherwise, you can show the entire bio.

Within the paragraph element, replace `bio` with a ternary operator. For the condition, check if the length of `bio` is greater than 50. If it is, use the `.slice()` method to extract the first 50 characters of `bio` and add an ellipsis at the end. Otherwise, show the full `bio`.
```